cmd/ddev/cmd/debug-compose-config.go

```diff
@@ -12,8 +13,9 @@ import (
 
 // DebugComposeConfigCmd implements the ddev debug compose-config command
 var DebugComposeConfigCmd = &cobra.Command{
-	Use:   "compose-config [project]",
-	Short: "Prints the docker-compose configuration of the current project",
+	ValidArgsFunction: ddevapp.GetProjectNamesFunc("all", 1),
```

Comment:
I imagine you already explored to see if this could be handled somewhere other than each Cobra Cmd.